lib/osml/tile_server/ts_dataplane.ts

```diff
@@ -197,12 +197,48 @@ export class TSDataplaneConfig extends BaseConfig {
    */
   public CONTAINER_REPOSITORY: string;
 
+  /**
+   * The path to Dockerfile.tile_server to use to build the container.
+   * @default "docker/Dockerfile.tile_server"
+   */
+  public CONTAINER_DOCKERFILE: string;
+
   /**
    * The default API path to use if auth was configured.
    * @default "latest/viewpoints"
    */
   public API_DEFAULT_PATH: string;
 
+  /**
+   * Optional flag to determine whether to deploy the test-related components.
+   * @default false
+   */
+  public DEPLOY_TEST_COMPONENTS: boolean;
```

Comment:
Does this need to have `?` since this is an optional flag

Reply:
This is defaulted so doesn't require a ? flag